src/terminal/terminal.rs

```diff
@@ -493,8 +495,41 @@ where
         self.backend.size()
     }
 
+    /// Draw lines at the given vertical offset. The slice of cells must contain enough cells
+    /// for the requested lines. A slice of the unused cells are returned.
+    fn draw_lines<'a>(
```

Comment:
Move this below where it's called from

Reply:
Okay. That'll requires a new impl block, but that's fine.